builder/hyperv/common/driver.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
```

Comment:
I've made the original code available under MPL 2 (as well as Apache 2) and removed all of the copyright headers. Will that work for this?